docs/spec/index.html

```diff
@@ -1,740 +1,876 @@
+      a given time. The following section details rules for handling identifiers
+      and references to data to prevent issues with informational processing.
+    </p>
+    <section class="informative">
+      <h2>Identifiers</h2>
+      <p>
+        Identifiers in objects exchanged using this specification are
+        particularly important, not least of which in that they uniquely
+        identify an object or network transaction, but also, given the
+        asynchronous nature of this mode of data exchange, that the present an
+        area where various attacks could arise by sending of data with
+        identifiers already in use with malicious intent to confuse a receiver
+        of the data.
       </p>
-
       <section class="informative">
-        <h3>CCG Standards Stack</h3>
-        <img src="resources/standards-stack.png" alt="standards-stack" width="100%">
+        <h2>Presentations</h2>
         <p>
-          The above diagram illustrates how traceability-interop fits into the
-          larger family of W3C CCG standards.
-        </p>
+          Presentations in this spec are <a
+            href="https://w3c-ccg.github.io/traceability-vocab/#TraceablePresentation">Traceable
+            Presentrations</a> which contain
+          a few notable identifiers that can be used for correlations,
+          retrievals, and business rule for processing of data.
+          <br />
+          <br />
+          These identifiers are:
         <ul>
-          <li>
-            At the <em>Business</em> layer, supply chain actors are exchanging
-            data in support of exchanged goods and services.
-          </li>
-          <li>
-            The <em>Trust</em> layer is supported by actors signing and
-            verifying data as Verifiable Credentials.
-          </li>
-          <li>
-            The <em>Process</em> defined standardized APIs for how to exchange
-            Verifiable Credentials (VC-API) and Verifiable Presentations
-            (trace-interop).
-          </li>
-          <li>
-            <em>Linked Data</em> establishes concise semantic context of the
-            data exchanged, enabled with JSON-LD by the trace-vocab
-            specification.
-          </li>
-          <li>
-            <em>Semantics</em> are defined in a multitude of established
-            vocabularies and come in many forms, from narrowly specialized,
-            industry-vertical vocabularies, to broadly-applicable, generic
-            vocabularies.
-          </li>
+          <li>`TraceablePresentation.id` the unique ID of the presentation
+            itself, think of this as identifying the wire transaction</li>
+          <li>`TraceablePresentation.workflow.definition` the ID for the
+            workflow definition, or type of workflow that the presented
+            credentials belong to</li>
+          <li>`TraceablePresentation.workflow.instance` the ID for the unique
```

Comment:
The definition here needs work, instances are meant to capture "many parties exchanging data", not just 2 parties.

A presentations is a "2 party" interaction.

An instance is "several parties working together to exchange presentations that meet a workflow definition requirements".

The primary use of instance is to allow multiple presentations to be "grouped together as related".